### ISSUE TITLE
Release 3.3

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#1453]
This issue is for tracking the progress of the 3.3 release.

### Blockers / Regressions

- [x] #1434
- [x] #1441
- [x] #1451
- [x] #1459
- [x] #1460 
- [x] #1482
- [x] #1546
- [x] https://github.com/sass/sassc/issues/139

### Nice to haves

- [x] #1396
- [x] #1405
- [x] #1413
- [x] #1461

### RC1 todos

- [x]  Release notes (@xzyfer)
- [x]  Release LibSass (@xzyfer)
- [x]  Release SassC (@xzyfer)
- [x]  Release [perl bindings] [1] (@mgreter)

### Stable todos

- [x]  Release notes
- [x]  Release LibSass
- [x]  Release SassC
- [x]  Release [perl bindings] [1] (@mgreter)

/cc @mgreter @saper @am11

[1]: https://github.com/sass/perl-libsass/

**Comment by xzyfer** [comment]
This first RC looks to be immanent. I'll start on the release notes tonight.

**Comment by JohnAlbin** [comment]
Go, team! Go!

Looking forward to this release as it fixes a show-stopping bug for me. #1280

**Comment by slyoldfox** [comment]
Same here. Curious to see if this will solve https://github.com/sass/node-sass/issues/970

**Comment by xzyfer** [comment]
I've just noticed that some recent LibSass changes have broken SassC. This is a blocker form 3.3. I'll investigate when I have a some time.

https://github.com/sass/sassc/issues/139

**Comment by xzyfer** [comment]
@mgreter can we feature freeze now so we can ship the first RC and release notes?

**Comment by xzyfer** [comment]
I'll have the release notes and RC ready to go for Monday. We're still blocked on sass/sassc#139 however.

**Comment by xzyfer** [comment]
@mgreter I've completed the release notes. Feel free to ship the release when you've merged your outstanding PRs.

**Comment by xzyfer** [comment]
Argh a new blocker was just reported (#1546).

**Comment by xzyfer** [comment]
#1546 is resolved.
